src/routes/examples/react/simple-example.mdx

```diff
@@ -0,0 +1,177 @@
+      // GET/POST/DELETE/PATCH endpoints;
+      // e.g. GET/POST of https://some.example.com/api/v1/users
+      this.resource('users');
+
+      // ... add additional /api/v1 routes here ...
+
+      // all other XHR requests are passed through to
+      // their destinations
+      this.passthrough();
+    }
+  });
+};
+
+export { startMirage };
+```
+
+## A Simple Data Class
+
+Below is a trivial class that `fetch`es some data from the api endpoint described
+in the `mirage.js` configuration file above.
+
+```js
+// src/data.js
+
+export default class Data {
+  async allUsers() {
+    let response = await fetch('https://some.example.com/allUsers');
+    if (!response.ok) {
+      throw new Error(`Response not 2xx.  Status Code ${response.status}`);
+    }
+    return await response.json();
+  }
+
+  async lookupUser(userId) {
+    let response = await fetch(`https://some.example.com/api/v1/users/${userId}`);
+    if (!response.ok) {
+      throw new Error(`Response not 2xx.  Status Code ${response.status}`);
+    }
+    return await response.json();
+  }
+}
+```
+
+## Testing The Data Class
+
+This Jest class sets up and tears down a MirageJs server for each test that attempts
+to hit the RESTful API endpoint described in the `mirage.js` configuration from above.
+
+
+```js
+// src/data.test.js
+
+import Data from './data';
+import { startMirage } from './mirage';
+
+// ------------------------------------------------ Before/After
+
+let server; // use the `server` to create data for each test
+
+beforeEach(() => {
+  server = startMirage(); // start MirageJs before each test
+});
+
+afterEach(() => {
+  server.shutdown();  // shutdown the MirageJs server after each test
+});
+
+// ------------------------------------------------------- Tests
+
+it('will return no users when requesting `/allUsers`', done => {
+  new Data().allUsers().then(response => {
+    expect(response.users.length).toBe(0);
+    done();
+  })
+});
+
+it('will return five users created by the mirage factory when requesting `/allUsers`', done => {
+  server.createList('user', 5);
+
+  new Data().allUsers().then(response => {
+    expect(response.users.length)
+        .toBe(5);
+    done();
+  })
+});
+
+it('will throw 404 when specific user cannot be found', async () => {
+  await expect(new Data().lookupUser(-1))
+      .rejects
+      .toThrow('Response not 2xx.  Status Code 404')
+});
+
+it('will return the requested user and their factory created information', done => {
+  let factoryUser = server.create('user');
+
+  new Data().lookupUser(factoryUser.id).then(response => {
+    expect(response.user.email)
+        .toBe(factoryUser.email);
+    done();
+  });
+});
+```
+
+## Next Steps
+
+Feel free to expand on this example by:
+
+* Introducing [relationships](/docs/data-layer/relationships) between model entities
+* Explore the [serializers](/docs/data-layer/serializers) that ship with MirageJs or create your own
```

Comment:
This next steps section was just a way to conclude the example.  We could refer to Ryan's video too once it becomes available.

Mind you, we could also create a few more examples using fixtures, relationships, and alternate payloads (`JSON:API`)